TRADEMARKS.md

```diff
@@ -0,0 +1,231 @@
+- [Trademarks Subject to the Guidelines](#Trademarks-Subject-to-the-Guidelines)
+- [Universal Considerations for All Uses](#Universal-Considerations-for-All-Uses)
+- [Use for Software](#Use-for-Software)
+- [Use for Non-Software Goods and Services](#Use-for-Non-Software-Goods-an-Services)
+- [General Considerations About Trademarks and Their Use](#General-Considerations-About-Trademarks-and-Their-Use)
+- [Where to Get Further Information](#Where-to-Get-Further-Information)
+
+# Introduction
+
+This document, the "Policy," outlines the AmpliPi project's (the "Project") policy for the use of our trademarks. While our software is available under a free and open source software license, the copyright license does not include an implied right or license to use our trademark.
+
+The role of trademarks is to provide assurance about the quality of the products or services with which the trademark is associated. But because an open source license allows your unrestricted modification of the copyrighted software, we cannot be sure that your modifications to the software are ones that will not be misleading if distributed under the same name. Instead, this Policy describes the circumstances under which you may use our trademarks.
+
+In this Policy we are not trying to limit the lawful use of our trademarks, but rather describe for you what we consider the parameters of lawful use to be. Trademark law can be ambiguous, so we hope to provide enough clarity for you to understand whether we will consider your use licensed or non-infringing.
+
+The sections that follow describe what trademarks are covered by this Policy, as well as uses of the trademarks that are allowed without additional permission from us. If you want to use our trademarks in ways that are not described in this Policy, or if you are uncertain about any of the following information contained in this Policy, please see "Where to get further information" below for contact information. Any use that does not comply with this Policy or for which we have not separately provided written permission is not a use that we have approved.
+
+# Our Commitment to Open Source Principles
+
+We want to encourage and facilitate the use of our trademarks by the community, but do so in a way that still ensures that the trademarks are meaningful as a source and quality indicator for our software and the associated goods and services and continue to embody the high reputation of the software and the community associated with it. This Policy therefore tries to strike the proper balance between: 1) our need to ensure that our trademarks remain reliable indicators of the qualities that they are meant to preserve and 2) our community members' desire to be full participants in the Project. 
+
+# Trademarks Subject to the Guidelines
+
+### Our Trademarks
+
+This policy covers:
+
+1. Our word trademark (the "Word Marks"): AmpliPi
+2. Our logos ("the Logos"): AmpliPi logos
```

Comment:
'the' is included in the double quotes here, but not in the other two things in the list